packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -790,6 +791,18 @@ const addTestVarsToWranglerToml = async (projectPath: string) => {
 	}
 };
 
+const assertEitherTomlOrJson = (projectPath: string) => {
+	const wranglerTomlPath = join(projectPath, "wrangler.toml");
+	const wranglerJsonPath = join(projectPath, "wrangler.json");
+
+	if (existsSync(wranglerTomlPath) && existsSync(wranglerJsonPath)) {
+		expect(
+			false,
+			"Either wrangler.toml or wrangler.json should exist, but not both",
+		).toBe(true);
+	}
```

Comment:
using `const hasJsonConfig = existSync(join(projectPath, "wrangler.json")) || existSync(join(projectPath, "wrangler.jsonc"));`

```suggestion
		expect(
			existsSync(wranglerTomlPath) === hasJsonConfig,
			"Either wrangler.toml or wrangler.json should exist, but not both",
		).toBe(false);
```